Client/Core/Commands/MiscHandler.cs

```diff
@@ -20,11 +20,11 @@ public static partial class CommandHandler
 
             new Thread(() =>
             {
-                string tempFile = FileHelper.GetTempFilePath(".exe");
+                var tempFile = FileHelper.GetTempFilePath(".exe");
```

Comment:
Unrelated changes like this makes the codebase inconsistent... We have been using explicit variable declarations in the rest of the codebase.